src/sentry/models/release.py

```diff
@@ -120,6 +123,65 @@ def filter_to_semver(self):
         """
         return self.filter(major__isnull=False)
 
+    def filter_by_semver(
+        self, organization_id: int, operator: str, version: str
```

Comment:
should we validate that `operator` is in `OPERATOR_TO_DJANGO.values()`

Reply:
I don't want to just limit it to that, since this could be any Django operator and we might need the flexibility in the future. Django will reject any incorrect operators, so I think we should be safe enough.

I'll think about it as I use this method more though, we might end up having some validation that makes sense here.